.jshintrc

```diff
@@ -9,5 +9,5 @@
   "nonew": true,
   "strict": true,
   "unused": true,
-  "esversion": 3
+  "esversion": 6
```

Comment:
I have to use for of loop ( Line 265 ) which is present in esversion 6 to iterate over the result returned by matchAll() method in Line 262. Otherwise it is giving error as for of loop is not present in esversion 3. for of loop gives a better and cleaner way to iterate over the result.